src/mon/MgrMonitor.cc

```diff
@@ -146,10 +143,12 @@ void MgrMonitor::create_initial()
   }
   pending_map.always_on_modules = always_on_modules();
   pending_command_descs = mgr_commands;
-  dout(10) << __func__ << " initial modules " << pending_map.modules
-	   << ", always on modules " << pending_map.get_always_on_modules()
-           << ", " << pending_command_descs.size() << " commands"
+  dout(10) << __func__ << " initial enabled modules: " << pending_map.modules
```

Comment:
As discussed, the volumes module should still be kept always on and force the user to pass `--yes...` flag to disable an always on module.

Reply:
What's the benefit to this approach vs. putting it in the "initially enabled" modules"?

BTW, I think relating to this there should be better mechanisms to turn off functions in the volumes plugin that are likely causing issues in production (temporarily), such as the cloner/purge threads. There is a tracker ticket somewhere for that.

Reply:
Are we really sure it's just the async threads that cause the mgr to engage request throttles? I need to check back on the respective BZs, but the sheer number of mgr/volumes commands were also a contributing factory iirc.

Reply:
Thinking into this more: I think this is kind of a last resort when things get really messy. So, the user should first try to disabled the async threads (purge and cloner threads) and if th slowness (throttles) are still getting engaged by the manager, then the last resort is to disable the volumes plugin altogether.